pkg/gui/controllers/files_controller.go

```diff
@@ -748,6 +755,90 @@ func (self *FilesController) createStashMenu() error {
 	})
 }
 
+func (self *FilesController) openCopyMenu() error {
+	copyNameNoItemSelectionErr := utils.ResolvePlaceholderString(
+		self.c.Tr.FileCopyNoItemSelectedError,
+		map[string]string{"target": "name"},
+	)
+	copyNameItem := &types.MenuItem{
+		Label: self.c.Tr.CopyFileName,
+		OnPress: func() error {
+			name := self.context().GetSelectedItemId()
+			if name == "" {
```

Comment:
This shouldn't be possible. We should set disabledReasons for all commands in all cases where a command isn't applicable, so that we never have to return errors from handlers.

Reply:
I was honestly unsure if it was worth it adding this check (that yes, should be not necessary given the `disabledReason`), so I've added it as an "extra check". It's fine for me to remove it (and the others).